presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1204,6 +1204,11 @@ private void testColumnName(String columnName, boolean delimited)
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    protected String createTableSqlForTestColumnName(String tableName, String columnName)
```

Comment:
To unblock Kudu, you could change `testColumnName` to use CTAS instead of CT+I, as this works for Kudu.

Per comment in the test method, we should actually test both flows.
That would require Kudu to implement the INSERT for "norma" tables. I think @Praveen2112 is looking into this.